modules/preview-react/pill/lib/PillIconButton.tsx

```diff
@@ -1,102 +1,83 @@
 import React from 'react';
 
-import {focusRing, styled, StyledType, createSubcomponent} from '@workday/canvas-kit-react/common';
-
-import {SystemIcon, SystemIconProps} from '@workday/canvas-kit-react/icon';
+import {focusRing, createSubcomponent} from '@workday/canvas-kit-react/common';
+import {SystemIcon, SystemIconProps, systemIconStencil} from '@workday/canvas-kit-react/icon';
 import {usePillModel} from './usePillModel';
 import {xSmallIcon} from '@workday/canvas-system-icons-web';
-import {CanvasSystemIcon} from '@workday/design-assets-types';
-import {colors, space} from '@workday/canvas-kit-react/tokens';
 import {BaseButton, buttonStencil} from '@workday/canvas-kit-react/button';
+import {calc, createStencil, px2rem} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
+import {mergeStyles} from '@workday/canvas-kit-react/layout';
 
-export interface PillIconButtonProps extends Omit<SystemIconProps, 'icon'> {
-  /**
-   * The system icon rendered by the button
-   * @default `xSmallIcon`
-   */
-  icon?: CanvasSystemIcon;
+export interface PillIconButtonProps extends Partial<SystemIconProps> {
   /**
    * The aria label for the removable icon
    * @default 'remove'
    */
-  'aria-label'?: string;
+  'aria-label': string;
```

Comment:
Do we want to add something to the comments referencing this:
```Pill.IconButton` now requires `aria-label` as a prop. The default `aria-label="remove"` was removed to ensure consumers pass a valid value```

Reply:
make as optional